### PR TITLE
WRP-832:  Add an entry point for ui/FloatingLayer for parsing jsdoc to typescript

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
A `package.json` file was added to ui/FloatingLayer to correctly parse jsdoc to typescript

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
A new path to `FloatingLayerDecorator` had to be added to ui/Spinner unit tests to correctly refer the `FloatingLayerDecorator`

### Links
[//]: # (Related issues, references)
WRP-832

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Comment by hong6316** [comment]
@adrian-cocoara-lgp 
Please write about 'FloatingLayerDecorator' changes in 'Additional Considers'. 
It's not related to jsdoc-to-ts, so it's right to fill it out separately.

File: packages/ui/FloatingLayer/package.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "main": "FloatingLayer.js"
```
Comment (hong6316) [inline packages/ui/FloatingLayer/package.json:2]:
```suggestion
  "main": "index.js"
```

File: packages/ui/Spinner/tests/Spinner-specs.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import '@testing-library/jest-dom';
 import {render, screen} from '@testing-library/react';
 
 import {SpinnerBase} from '../Spinner';
-import {FloatingLayerDecorator} from '../../FloatingLayer';
+import {FloatingLayerDecorator} from '../../FloatingLayer/FloatingLayerDecorator';
```
Comment (hong6316) [inline packages/ui/Spinner/tests/Spinner-specs.js:5]:
```suggestion
import {FloatingLayerDecorator} from '../../FloatingLayer';
```

**Comment by hong6316** [comment]
to @adrian-cocoara-lgp ,
Please modify these 2 changes and re-test again.
We do not agree that we need to fix moonstone/sandstone/agate after merging this PR.

**Pull request merged**